charts/gateway/templates/deployment.yaml

```diff
@@ -463,6 +468,14 @@ spec:
             items:
             - key: license
               path: license.xml
+  {{- if not .Values.disklessConfig.enabled }}
+      - name: {{ template "gateway.fullname" . }}-node-properties
+        secret:
+          secretName: {{ template "gateway.node.properties" . }}
+          items:
+            - key: node.properties
+              path: node.properties
```

Comment:
The indents are off... I'll fix this after I test

Reply:
Addressed in https://github.com/CAAPIM/apim-charts/pull/318/commits/0d5189e9571af9724fe57b2d231e71104ec93812